cmake/modules/LimitJobs.cmake

```diff
@@ -4,13 +4,18 @@ set(MAX_LINK_MEM 4500 CACHE INTERNAL "maximum memory used by each linking job (i
 cmake_host_system_information(RESULT _num_cores QUERY NUMBER_OF_LOGICAL_CORES)
 cmake_host_system_information(RESULT _total_mem QUERY TOTAL_PHYSICAL_MEMORY)
 
-math(EXPR _avg_compile_jobs "${_total_mem} / ${MAX_COMPILE_MEM}")
-if(_avg_compile_jobs EQUAL 0)
-  set(_avg_compile_jobs 1)
-endif()
-if(_num_cores LESS _avg_compile_jobs)
-  set(_avg_compile_jobs ${_num_cores})
+if (FORCE_MAX_JOBS)
```

Comment:
could you please remove the space after `if` to be more consistent with the other part of this CMake script?